pkg/guacanalytics/patchPlanning.go

```diff
@@ -190,3 +200,26 @@ func exploreIsDependencyFromDepPkg(ctx context.Context, gqlclient graphql.Client
 
 	return nil
 }
+
+// TODO: impelement this functions
+func exploreIsOccurrence(ctx context.Context, gqlclient graphql.Client, q *queueValues, isOccurrence model.NeighborsNeighborsIsOccurrence) error {
+	// Step 1: Find Artifact attached to package through IsOccurence
+	// -> call .Artifact on isOccurrence
```

Comment:
Do we need to call `query Artifact` here?

Reply:
I believe if I simply input isOccurrence.Artifact.Id into .Neighbors that would be sufficent to access the artifact we need.

Reply:
why do we need to call `Artifact` here though?

Reply:
I want to call .Artidact so I can reach the Artifact so that we can find any HasSLSAs it is connected to, to find the relevent affected packages